queries/yuck/highlights.scm

```diff
@@ -0,0 +1,66 @@
+(ERROR) @error
+
+(line_comment) @comment
+
+; keywords and symbols
+
+(keyword) @keyword
+(symbol) @function
```

Comment:
function is probably correct here (I don't know the language). Just wanted to make aware of `@symbol`

Reply:
Well actually I used `@tag` in helix. `symbol` is most of the time *some* kind of DOM/graphical element, like `box` or `defwidget` etc. but not all the time, there is e.g. a `for` symbol. So I don't know maybe also use `@tag`, I'm not sure?